docs/source/changes.rst

```diff
@@ -15,10 +15,19 @@ Announcements
    so that it can be included as part of the ReadTheDocs documentation
    `here <https://pyquil.readthedocs.io/en/stable/changes.html>`__
    (@karalekas, gh-945).
+-  Deprecation warnings have been added regarding the ``fCPHASE`` and
+   ``fBellState`` device spec fields, which are no longer routinely
+   updated and will be removed in a coming release.
 
 Improvements and Changes
 ~~~~~~~~~~~~~~~~~~~~~~~~
 
+- In addition to the simultaneous gate operation 1Q RB fidelities
```

Comment:
please move these entries to CHANGELOG.md instead, and add (@jvalery2, gh-968) to the end